lib/cocoapods/generator/app_target_helper.rb

```diff
@@ -0,0 +1,136 @@
+module Pod
```

Comment:
this is extracted out from `validator.rb` and provides a bunch of helper methods, similar to `xcconfig_helper.rb`.

Easier to test and I can reuse this logic when I create app host target support below.